velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -48,38 +48,81 @@ static std::vector<std::string> kSortOrderBasedOverClauses = {
     "order by c1 desc nulls first, c0 asc nulls first",
 };
 
-/// Common set of window function over clauses for window frame tests to ensure
-/// total ordering for deterministic results in ROW mode.
-static std::vector<std::string> kFrameOverClauses = {
-    "partition by c0 order by c1, c2",
-    "partition by c1 order by c0, c2",
+/// Common set of window function over clauses using a combination of three
+/// columns, that produces a deterministic, total ordering of all rows.
+static std::vector<std::string> kOrderedBasicOverClauses = {
+    "partition by c0 order by c1 desc, c2",
+    "partition by c1 order by c0, c2 desc",
     "partition by c0 order by c2, c1",
-    "partition by c1 order by c2, c0",
+    "partition by c1 order by c2, c0 desc",
     // No partition by clause.
-    "order by c0 asc nulls first, c1 desc, c2 asc",
+    "order by c0 asc nulls first, c1 desc, c2",
     "order by c1 asc, c0 desc nulls last, c2 desc",
     "order by c0 asc, c2 desc, c1 asc nulls last",
-    "order by c2 asc, c1 desc nulls first, c0 asc",
+    "order by c2 asc, c1 desc nulls first, c0",
     // No order by clause.
     "partition by c0, c1, c2",
 };
 
-/// Common set of window function frame clauses in RANGE mode, with current row,
-/// unbounded preceding, and unbounded following frame combinations.
-static std::vector<std::string> kRangeFrameClauses = {
+/// Common set of window function over clauses with different sort orders
+/// using a combination of three columns, that produces a deterministic, total
+/// ordering of all rows.
+static std::vector<std::string> kOrderedSortOrderBasedOverClauses = {
```

Comment:
How are these different from kOrderedBasicOverClauses?

> kOrderedSortOrderBasedOverClauses

Having both Ordered and SortOrder in the name feels redundant.

Reply:
@pramodsatya : Why is this new list needed ? The only interesting combination is partition by multiple columns that isn't in the first list.

Reply:
It was initially added to introduce total ordering of rows in kSortOrderBasedOverClauses without modifying the existing list. Removed the duplicate lists and retained the list of ordered clauses.